src/Microsoft.ML.Data/Dirty/ChooseColumnsByIndexTransform.cs

```diff
@@ -191,9 +139,9 @@ private static VersionInfo GetVersionInfo()
         {
             return new VersionInfo(
                 modelSignature: "CHSCOLIF",
-                verWrittenCur: 0x00010001, // Initial
-                verReadableCur: 0x00010001,
-                verWeCanReadBack: 0x00010001,
+                verWrittenCur: 0x00010002,
```

Comment:
>0x00010002 [](start = 31, length = 10)

I would say, if you can preserve old saving mechanism, try to preserve it.
Do you have any particular reason to switch to new format?

Reply:
This format has been changed previously. You can see that the old doc string describing doesn't match what previously stored.

---
In reply to: [241838591](https://github.com/dotnet/machinelearning/pull/1879#discussion_r241838591) [](ancestors = 241838591)

Reply:
I've check internal repo, and format is same as in code you changing.

---
In reply to: [241843452](https://github.com/dotnet/machinelearning/pull/1879#discussion_r241843452) [](ancestors = 241843452,241838591)